### ISSUE TITLE
Does not support aliases

### ISSUE THREAD

**Issue by deniss-s** [AsaAyers/js-hyperclick#78]
We have babel.rc in our project with aliases
```
"plugins": [
		"transform-decorators-legacy",
		"transform-class-properties",
		["module-resolver", {
			"root": ["./src"],
			"alias": {
				"app": "js",
				"api": "js/api",
			}
		}]
	]
```
I can solve this partially. E. g. imports like `import {SomeApi} from 'api'` by specifying `moduleRoots` in package.json, but `import {myFunc} from 'app/utils'` won't work, since app is an aliase for js.

**Comment by AsaAyers** [comment]
See https://github.com/AsaAyers/js-hyperclick/issues/58

**Comment by AsaAyers** [comment]
I just published v1.15 with a new feature I'm calling "Custom Resolvers". It will allow you to add a JS file to your project that can implement your aliases. [PR and documentation](https://github.com/AsaAyers/js-hyperclick/pull/105)
